### PR TITLE
Added default probe settings for ca

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: install/kubernetes/templates/istio-ca.yaml.tmpl [hunk 1]
```diff
@@ -35,7 +35,19 @@ spec:
           - --grpc-port=8060
           - --grpc-hostname=istio-ca
           - --self-signed-ca=true
+          - --liveness-probe-path=/usr/local/bin/ca.liveness # path to the liveness health check status file
```
Comment (myidpt) [inline install/kubernetes/templates/istio-ca.yaml.tmpl]: Can you change it to /tmp/ca.liveness?

File: install/kubernetes/templates/istio-ca.yaml.tmpl [hunk 2]
```diff
@@ -35,7 +35,19 @@ spec:
...
           - --logtostderr
           - --stderrthreshold
           - INFO
+        livenessProbe:
+          exec:
+            command:
+            - /usr/local/bin/istio_ca
+            - probe
+            - --probe-path=/usr/local/bin/ca.liveness  # path to the liveness health check status file
+            - --interval=125s                          # maximum duration that passed since last check before health check restart the pod
```
Comment (myidpt) [inline install/kubernetes/templates/istio-ca.yaml.tmpl]: Maybe change it to:
\# the maximum time gap allowed between the file mtime and the current sys clock.

**Comment by myidpt** [comment]
Thanks a ton for the work!
Can we put this config to a single template:
install/kubernetes/templates/istio-ca-with-health-check.yaml.tmpl (I think in this PR you just need to create this file).
And then we can put together these templates with health check into a file:
isito-auth-with-control-plane-health-check.yaml.

File: install/kubernetes/templates/istio-ca.yaml.tmpl (hunk 2 above)
Comment (mangchiandjjoe) [reply install/kubernetes/templates/istio-ca.yaml.tmpl]: Updated

File: install/kubernetes/templates/istio-ca.yaml.tmpl (hunk 1 above)
Comment (mangchiandjjoe) [reply install/kubernetes/templates/istio-ca.yaml.tmpl]: Renamed

**Pull request merged**